index.html

```diff
@@ -2793,12 +2774,11 @@ <h3><dfn>Writing or pushing content</dfn></h3>
             </dl>
           </li>
           <li>
-            Let |name:string| be the result of running the
-            <a>validate external type</a> steps on |record|'s
-            <a>recordType</a>. If |name| is not `false`,
+            If running <a>validate external type</a> on |record|'s
+            <a>recordType</a> returns |type| that is not `false`,
```

Comment:
the question here is, is the validation algo transforming the input (the steps from parse-host do, but do we use it, since it comes from apps)? If the steps don't change the record type, the algorithm can just succeed or fail, and no need to return `false` or a transformed string.